src/v/pandaproxy/schema_registry/handlers.cc

```diff
@@ -516,4 +516,20 @@ compatibility_subject_version(server::request_t rq, server::reply_t rp) {
     co_return rp;
 }
 
+ss::future<server::reply_t>
+get_restart(server::request_t rq, server::reply_t rp) {
+    parse_accept_header(rq, rp);
+    rq.req.reset();
+
+    vlog(plog.info, "restart internal store");
+
+    auto client_cfg = config::to_yaml(
+      rq.service().client().local().config(), config::redact_secrets::no);
+    co_await rq.service().client().stop();
+    co_await rq.service().schema_store().clear();
+    co_await rq.service().client().start(client_cfg);
```

Comment:
I wonder if `start()` is allowed after `stop()` on a client? Not sure it was designed to do that.

Reply:
Yeah I took a look, doesn't look like it is allowed.